2.sequential-compression/README.md

```diff
@@ -1,11 +1,11 @@
-# Ensemble Z Analysis
+# Sequential Compression Analysis
 
 **Gregory Way, 2018**
 
 Gene expression data compression reveals coordinated gene expression modules that describe important biology.
 
-In the following analysis, we apply five compression algorithms to serially compress three gene expression datasets across various bottleneck dimensionalities.
-We save the population of all models, for each algorithm, across z for downstream analyses.
+In the following analysis, we apply five compression algorithms to sequentially compress three gene expression datasets across various bottleneck dimensions (_k_).
```

Comment:
Wording is a bit confusing to me here.  "We compress 5 instances/runs of a given compression algorithm"

Reply:
yeah, this is confusing. Will reword!